src/io.h

```diff
@@ -75,11 +77,13 @@ typedef struct io_ops {
    io_cleanup_all_fn         cleanup_all;
    io_thread_register_fn     thread_register;
    io_max_latency_elapsed_fn max_latency_elapsed;
+   io_get_context_fn         get_context;
```

Comment:
Added helper `get` method to return IO-Context. 

This is used in tests. (Currently, it's the same for all threads, but in some new development, this IO-context could be thread-specific. In that scenario, this kind of `get` method helps clarify the testing logic.)